### PR TITLE
Add billable items API entry point documentation

### PR DESCRIPTION
**Pull request by bpinto**

### CODE REVIEW

File: sections/billable_items.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+**Example Response**
+```json
+{
+  "billable items": [
+    {
+      "id": 1,
+      "item_code": "0001",
+      "item_type": "Service"
```
Comment (mphalliday) [inline sections/billable_items.md]: ,

File: sections/billable_items.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  -H 'Accept: application/json' \
+  -H 'User-Agent: APP_VENDOR_NAME (APP_VENDOR_EMAIL)'
+```
+
+**Example Response**
+```json
+{
+  "id": 1,
+  "item_code": "0001",
+  "item_type": "Service"
```
Comment (mphalliday) [inline sections/billable_items.md]: Also here `,`

File: sections/billable_items.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+```
+**Example Response**
+```
+Headers { Location: http://api.cliniko.com/billable items/2 }
+```
+```json
+{
+  "id": 1,
+  "item_code": "0002",
+  "item_type": "Service"
```
Comment (mphalliday) [inline sections/billable_items.md]: And here

File: sections/billable_items.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+  -H 'User-Agent: APP_VENDOR_NAME (APP_VENDOR_EMAIL)' \
+  -d '{ "name": "Billable Item 1" }' \
+  -X PUT
+```
+**Example Response**
+```json
+{
+  "id": 1,
+  "item_code": "0001",
+  "item_type": "Service"
```
Comment (mphalliday) [inline sections/billable_items.md]: And here :smile:

**Comment by bpinto** [comment]
Fixed! :dancer:

**Comment by mphalliday** [comment]
:dancers::dancers::dancers:

**Pull request merged**